components/form/style/index.less

```diff
@@ -295,6 +295,14 @@
       }
     }
 
+    // fixes https://github.com/ant-design/ant-design/issues/20482
+    .@{ant-prefix}-input-group-addon .@{ant-prefix}-select {
+      &.@{ant-prefix}-select-single:not(.@{ant-prefix}-select-customize-input)
+        .@{ant-prefix}-select-selector {
+        border: 0;
+      }
+    }
```

Comment:
Remove this line: https://github.com/ant-design/ant-design/blob/7998a3312e780cec232f6b1d5ad389c6f9a63288/components/form/style/index.less#L305

Reply:
Hey @zombieJ , sorry I didn't understand.  Did you mean to remove the `&-selector` on `L305` and add the class name for `selector` like below ?
```
.@{ant-prefix}-input-group-addon .@{ant-prefix}-select .@{ant-prefix}-select-selector {
        border-color: transparent;
        box-shadow: none;
  }
```

Reply:
I mean `L305` is a fix for this in v3 but not work in v4. If you add new patch to fix this, old one should be removed.

Reply:
Cool thanks 👍 Have pushed the fix